packages/manager/modules/vps/src/backup-storage/index.js

```diff
@@ -1,12 +1,24 @@
+import angular from 'angular';
+
 import component from './vps-backup-storage.component';
 import routing from './vps-backup-storage.routing';
 
 import ovhManagerVpsBackupStorageOrder from './order';
+import addStorage from './add';
+import deleteStorage from './delete';
+import editStorage from './edit';
+import password from './password';
 
 const moduleName = 'ovhManagerVpsBackupStorage';
 
 angular
-  .module(moduleName, [ovhManagerVpsBackupStorageOrder])
+  .module(moduleName, [
+    ovhManagerVpsBackupStorageOrder,
```

Comment:
àlphabetical order

also, you use routing, translations and ui-kit, add those dependencies too